portfolio/src/main/java/com/google/sps/data/CommentData.java

```diff
@@ -0,0 +1,31 @@
+//Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Date;
+
+// Class containing comment data
+public class CommentData {
+ 
+  // List of comments
+  private final ArrayList<String> comments = new ArrayList<String>();
+ 
+   public void logComment(String userName, String userComment) {
+        comments.add(userName + ": " + userComment);
```

Comment:
This works for now, but it might make things harder if this was something you wanted to build on in the long run. For example, combining the userName and comment together into a string might make it harder to find all the comments by a particular person or to present an attribution free version of the comments. Definitely not something you need to change, just something to think about as you design things like this.